guides/InstallationAdminGuide.md

```diff
@@ -333,6 +333,8 @@ Once the server has been started, the success of the installation can be checked
 
 The openEQUELLA Server administration account is hidden from casual users and is displayed by entering a special URL created from the server’s base URL. 
 
+Note: The base URL is configured in `learningedge-config/mandatory-config.properties` as `admin.url`. If admin.url is changed adjust the instructions below accordingly.
+
```

Comment:
I would change:

The openEQUELLA Server administration account is hidden from casual users and is displayed by entering a special URL created from the server’s base URL. 
Note: The base URL is configured in `learningedge-config/mandatory-config.properties` as `admin.url`. If admin.url is changed adjust the instructions below accordingly.

to

The openEQUELLA Server administration account is hidden from casual users and is displayed by entering a special URL.  This URL is configured in `learningedge-config/mandatory-config.properties` as `admin.url`.

Reply:
I have applied the suggested wording in f3942e7